### DISCUSSION TITLE
default python versions break precommit workflows

### DISCUSSION THREAD

**Discussion by devkral** [dymmond/esmerald#314] [General]
When specified in precommit it prevents users with other python versions to use precommit hooks.

I think it mandates the python version for all workflows used by default (can be overwritten in workflows itselfs)

**Comment by tarsil** [comment]
Did you have any problems because of the python version there?

**Reply by devkral** [reply]
yes, it tries to use python 3.10 but I have python 3.12 (anyway, see the changes)

**Comment by tarsil** [comment]
Changed before
